packages/@aws-cdk-testing/framework-integ/test/aws-route53-targets/test/integ.appsync-target.ts

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+import * as appsync from 'aws-cdk-lib/aws-appsync';
+import * as acm from 'aws-cdk-lib/aws-certificatemanager';
+import * as route53 from 'aws-cdk-lib/aws-route53';
+import { App, Stack } from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import * as targets from 'aws-cdk-lib/aws-route53-targets';
+import { join } from 'path';
+
+type TestStackProps = {
+  certificateArn: string;
+  domainName: string;
+  hostedZoneId: string;
+}
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props: TestStackProps) {
+    super(scope, id);
+
+    const { domainName, certificateArn, hostedZoneId } = props;
+
+    const certificate = acm.Certificate.fromCertificateArn(this, 'cert', certificateArn);
+
+    const api = new appsync.GraphqlApi(this, 'api', {
+      definition: appsync.Definition.fromFile(join(__dirname, '..', '..', 'aws-appsync', 'test', 'appsync.test.graphql')),
+      domainName: {
+        certificate,
+        domainName,
+      },
+      name: 'api',
+    });
+
+    const zone = route53.HostedZone.fromHostedZoneAttributes(this, 'hosted-zone', {
+      zoneName: domainName,
+      hostedZoneId,
+    });
+
+    new route53.ARecord(this, 'Alias', {
+      zone,
+      target: route53.RecordTarget.fromAlias(new targets.AppSync(api)),
+    });
+  }
+}
+
+const certificateArn = process.env.CDK_INTEG_CERTIFICATE_ARN ?? process.env.CERTIFICATE_ARN;
```

Comment:
```suggestion
const certificateArn = process.env.CDK_INTEG_CERT_ARN ?? process.env.CERTIFICATE_ARN;
```

It seems that the environment variable name used by integ-runner is incorrect.

https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/integ-runner/lib/runner/runner-base.ts#L431

Reply:
Good catch. I've adjusted within 73a4a46fdc. This could be why the PR CodeBuild was failing too.